dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -442,7 +442,17 @@ class LocalAdmissionController final : private boost::noncopyable
         background_threads.emplace_back([this] { this->watchGAC(); });
     }
 
-    ~LocalAdmissionController() { stop(); }
+    ~LocalAdmissionController()
+    {
+        try
+        {
+            stop();
+        }
+        catch (...)
+        {
+            LOG_ERROR(log, "stop server id({}) failed: {}", unique_client_id, getCurrentExceptionMessage(false));
```

Comment:
I'm ok with this fix. But actually we need to find out the root cause why grpc throw this exception.

Because the last grpc(final_report of LAC) have to succeed, otherwise the function of resource control may be affected.

Reply:
> I'm ok with this fix. But actually we need to find out the root cause why grpc throw this exception.
> 
> Because the last grpc(final_report of LAC) have to succeed, otherwise the function of resource control may be affected.

The exception says: failed to connect to all addresses. So I think this may be a network error.

Reply:
If PD is shutdowning, it will be ok.